### DISCUSSION TITLE
Updating Source Trigger cannot be compiled

### DISCUSSION THREAD

**Discussion by qfmee** [AvaloniaUI/Avalonia#16726] [General]
Xaml:
`<NumericUpDown Value="{Binding PositionOfSpotToBase, UpdateSourceTrigger=PropertyChanged}"  />`

Compile error:
AVLN2000	Unable to resolve suitable regular or attached property UpdateSourceTrigger on type Avalonia.Markup.Xaml:Avalonia.Markup.Xaml.MarkupExtensions.ReflectionBindingExtension 

Why can't I use this feature? Have you encountered the same problem?

Windows 11
Visual studio 2022
Avalonia Version="11.1.1"
Avalonia for Visual Studio 11.7

**Comment by ILD3225** [comment]
Hello.
Try to enable the option "Use compiled bindings by default" 
![image](https://github.com/user-attachments/assets/dd903c3d-3d32-44b3-a768-5725ecc7f07b)
 or use Compiledbinding
![image](https://github.com/user-attachments/assets/b5c24947-44d8-477e-8f7a-1ba23c411a8b)

**Comment by timunie** [comment]
@qfmee are you sure you are using 11.1 version of Avalonia? Older versions don't support this feature.
